.gitmodules

```diff
@@ -190,4 +190,7 @@
 	url = https://github.com/conpdatasets/MB-Cholinergic-deficient-mouse-model.git
 [submodule "projects/mousebytes/MB-Co-transmission-of-cholinergic-interneurons"]
 	path = projects/mousebytes/MB-Co-transmission-of-cholinergic-interneurons
-	url = https://github.com/conpdatasets/MB-Co-transmission-of-cholinergic-interneurons.git
+	url = https://github.com/Rodent-Cognition-Core/MB-Co-transmission-of-cholinergic-interneurons.git
```

Comment:
```suggestion
	url = https://github.com/conpdatasets/MB-Co-transmission-of-cholinergic-interneurons.git
```

Reply:
sorry, I did not resolve the conflict correctly. See suggested changes above to resolve that.